apps/fulfillment-e2e/src/support/index.ts

```diff
@@ -10,3 +10,14 @@
 
 // Import commands.js using ES2015 syntax:
 import './commands';
+
+Cypress.on('uncaught:exception', (err) => {
+  const ignoreErrors = [
+    'Registration failed - push service not available',
+    'Background Sync is disabled',
+  ];
+
+  if (ignoreErrors.some((ignoreError) => err.message.includes(ignoreError))) {
+    return false;
```

Comment:
After the call with @AndreyYevtukhov we decided to extract e2e tests into [separate task](https://spryker.atlassian.net/browse/HRZ-2946) because we didn't find a proper way to "Allow" push notifications in headless mode.